msupply-foundation/open-msupply
# 3186 delete inbound return

Fixes #3186 

# 👩🏻‍💻 What does this PR do? 
 

Implements API for deleting an Inbound Return.

# 🧪 How has/should this change been tested? 


Automated tests in service layer.

- From inbound returns list view
- Select some returns
- [ ] Delete selected lines, they should be removed
- From inbound return detail page, of a NEW return
- Open the side panel
- [ ] Click delete button, return should be deleted

## 💌 Any notes for the reviewer?


Using #3278 as a base as it has the deleteStockInLine abstraction.
